mat/format.go

```diff
@@ -51,18 +54,35 @@ func DotByte(b byte) FormatOption {
 	return func(f *formatter) { f.dot = b }
 }
 
-// Squeeze sets the printing behaviour to minimise column width for each individual column.
+// Squeeze sets the printing behavior to minimise column width for each individual column.
 func Squeeze() FormatOption {
 	return func(f *formatter) { f.squeeze = true }
 }
 
+// FormatMATLAB sets the printing behavior to output MATLAB syntax. If MATLAB syntax is
+// specified, the ' ' verb flag and Excerpt option are ignored. If the alternative syntax
+// verb flag, '#' is used the matrix is formatted in rows and columns.
+func FormatMATLAB() FormatOption {
+	return func(f *formatter) { f.format = formatMATLAB }
+}
+
+// FormatPython sets the printing behavior to output Python syntax. If Python syntax is
+// specified, the ' ' verb flag and Excerpt option are ignored. If the alternative syntax
+// verb flag, '#' is used the matrix is formatted in rows and columns.
+func FormatPython() FormatOption {
+	return func(f *formatter) { f.format = formatPython }
+}
+
 // Format satisfies the fmt.Formatter interface.
 func (f formatter) Format(fs fmt.State, c rune) {
-	if c == 'v' && fs.Flag('#') {
+	if c == 'v' && fs.Flag('#') && f.format == nil {
 		fmt.Fprintf(fs, "%#v", f.matrix)
 		return
 	}
-	format(f.matrix, f.prefix, f.margin, f.dot, f.squeeze, fs, c)
+	if f.format == nil {
+		f.format = format
```

Comment:
perhaps rename `func format(...)` into `func formatGo(...)` (or `formatGonum`) so everything is symetric?

Reply:
There's not an underlying symmetry, so I don't think this is worth doing.